packages/cli/src/commands/validator/status.ts

```diff
@@ -8,19 +8,30 @@ import { BaseCommand } from '../../base'
 import { newCheckBuilder } from '../../utils/checks'
 import { Flags } from '../../utils/command'
 
+export const statusTable = {
+  address: {},
+  name: {},
+  signer: {},
+  elected: {},
+  frontRunner: {},
+  proposed: {},
+  signatures: {},
+  signatures: { get: (v: Validator) => parseInt(v.signatures * 100) + '%' },
+}
+
 export default class ValidatorStatus extends BaseCommand {
   static description =
     'Show information about whether the validator signer is elected and validating. This command will check that the validator meets the registration requirements, and its signer is currently elected and actively signing blocks.'
 
   static flags = {
     ...BaseCommand.flags,
-    signer: Flags.address({
-      description: 'address of the signer to check if elected and validating',
-      exclusive: ['validator'],
-    }),
```

Comment:
Removing this flag will mean that you can't check the status of a signer that is no longer the authorized signer, but is still part of the elected validator set until the end of the epoch. The intention is to allow people who are running two validator nodes as part of the key rotation process to know when they can safely take the rotated out signer offline.

Reply:
Oh I love that! Sorry for not understanding, added this functionality back